frontend/src/modules/Omics/views/OmicsRunCreateForm.js

```diff
@@ -151,7 +64,7 @@ const OmicsRunCreateForm = (props) => {
           workflowUri: omicsWorkflow.workflowUri,
           parameterTemplate: values.parameterTemplate,
           SamlAdminGroupName: values.SamlAdminGroupName,
-          destination: values.destination
```

Comment:
similar to importDashboard - can we keep the key value of `destination` for `createOmicsRun` the same